packages/nextjs/src/index.server.ts

```diff
@@ -20,6 +19,17 @@ export { ErrorBoundary, withErrorBoundary } from '@sentry/react';
 type GlobalWithDistDir = typeof global & { __rewriteFramesDistDir__: string };
 const domain = domainModule as typeof domainModule & { active: (domainModule.Domain & Carrier) | null };
 
+// During build, the main process is invoked by
+//   `node next build`
+// and child processes are invoked as
+//   `node <path>/node_modules/jest-worker/build/workers/processChild.js`,
+// whereas at runtime the process is invoked as
+//   `node next start`
+// or
+//   `node /var/runtime/index.js`.
+const isBuild = new RegExp('build').test(process.argv.toString());
```

Comment:
The pattern is a fixed string, there's no need to use a RegExp -- we could use `String.indexOf` or `String.includes`.

Also, `process.argv.toString()` seems brittle -- this could produce the incorrect output if "build" appears somewhere on the path / args by chance.

```suggestion
const isBuild = process.argv.toString().indexOf('build') >= 0;
```

I could give a better suggestion once I understand what exactly was the intent here (are we trying to cover the first and second cases of the comment?).